insalan/tournament/admin.py

```diff
@@ -151,6 +407,31 @@ def __init__(self, *args, **kwargs):
                 "content_type"
             )
```

Comment:
It's hard to know only with the name which slot you are picking. One thing that could improve this is to only show slots in the current tournament. You can achieve it just by filtering the queryset :
```python
seat_slot = self.fields.get("seat_slot")
if (seat_slot and self.instance.tournament):
    seat_slot.queryset = seat_slot.queryset.filter(
        tournament=self.instance.tournament
    )
```